scripts/scil_bingham_visualize.py

```diff
@@ -20,8 +20,13 @@
                              add_verbose_arg,
                              assert_inputs_exist,
                              assert_outputs_exist)
-from scilpy.viz.scene_utils import (create_bingham_slicer,
-                                    create_scene, render_scene)
+from scilpy.utils.util import RAS_AXES_NAMES, get_axis_index
+# TODO: There should not be as less backend in scripts as possible
```

Comment:
Not sure I understand the sentence.

Reply:
I removed it, it was more a development comment than anything